[2025-03-02T20:00:42.298+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-02T20:00:42.315+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.clean_data scheduled__2025-03-02T19:00:00+00:00 [queued]>
[2025-03-02T20:00:42.324+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.clean_data scheduled__2025-03-02T19:00:00+00:00 [queued]>
[2025-03-02T20:00:42.325+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-02T20:00:42.343+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): clean_data> on 2025-03-02 19:00:00+00:00
[2025-03-02T20:00:42.351+0000] {standard_task_runner.py:72} INFO - Started process 713 to run task
[2025-03-02T20:00:42.356+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline', 'clean_data', 'scheduled__2025-03-02T19:00:00+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpz5tldm2z']
[2025-03-02T20:00:42.360+0000] {standard_task_runner.py:105} INFO - Job 23: Subtask clean_data
[2025-03-02T20:00:42.388+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-02T20:00:42.422+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.clean_data scheduled__2025-03-02T19:00:00+00:00 [running]> on host 150a7fb033c6
[2025-03-02T20:00:42.425+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-02T20:00:42.510+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='clean_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-02T19:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-02T19:00:00+00:00'
[2025-03-02T20:00:42.512+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-02T20:00:42.530+0000] {data_pipeline.py:51} INFO - Executing script: /opt/airflow/scripts/data_preprocessing.py
[2025-03-02T20:02:27.299+0000] {data_pipeline.py:53} INFO - Successfully executed: /opt/airflow/scripts/data_preprocessing.py
[2025-03-02T20:02:27.300+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-02T20:02:27.307+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-02T20:02:27.308+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=clean_data, run_id=scheduled__2025-03-02T19:00:00+00:00, execution_date=20250302T190000, start_date=20250302T200042, end_date=20250302T200227
[2025-03-02T20:02:27.332+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-02T20:02:27.352+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-02T20:02:27.356+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
